lib/pangea/controllers/class_controller.dart

```diff
@@ -156,4 +158,24 @@ class ClassController extends BaseController {
     // BE - check class code and if class code is correct, invite student to room
     // FE - look for invite from room and automatically accept
   }
+
+  Future<void> addMissingRoomRules(String? roomId) async {
+    if (roomId == null) return;
+    final Room? room = _pangeaController.matrixState.client.getRoomById(roomId);
```

Comment:
@ggurdin Hey it would be cleaner to make this a class method of Room via pangea_room_extension. I'll go ahead and merge but keep in mind for the future!